### PR TITLE
Removes methods that were just rippled wrappers

### PR DESCRIPTION
**Pull request by mvadari**
## High Level Overview of Change

This PR removes all methods from the client that were just wrappers of rippled methods, and didn't really add anything else.

### Context of Change

xrpl.js 2.0

### Type of Change

- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Test Plan

CI passes. Tests that tested these methods were deleted.

### CODE REVIEW

**Comment by intelliot** [comment]
Nice - it will be interesting to see how much smaller the library gets, with this cruft taken out.

**Review by mukulljangid — APPROVED** [review]

File: snippets/src/getTransaction.ts [hunk 1]
```diff
@@ -1,15 +1,19 @@
...
+  const ledger = await client.request({command: 'ledger', transactions: true})
   console.log(ledger)
-  const tx = await client.getTransaction(ledger.transactionHashes[0])
+  const tx = await client.request({
+    command: 'tx',
+    transaction: ledger.result.ledger.transactions[0] as string
+  })
   console.log(tx)
-  console.log('deliveredAmount:', tx.outcome.deliveredAmount)
+  console.log('deliveredAmount:', (tx.result.meta as TransactionMetadata).DeliveredAmount)
```
Comment (natenichols) [inline snippets/src/getTransaction.ts:17]: Shouldn't `ledger` be of type `LedgerResponse`, and thus have metadata w/o needing the typecast?

Comment (mvadari) [reply snippets/src/getTransaction.ts:17]: It's possible for `meta` to be a string, if you ask for it to be returned in binary format

**Review by natenichols — APPROVED** [review]
👍

File: snippets/src/getTransaction.ts (hunk 1 above)
Comment (natenichols) [reply snippets/src/getTransaction.ts:17]: Ah gotcha, then can we check and not have to type cast.

```ts
if (typeof tx.result.meta !== 'string') {
    console.log(...)
}
```

Comment (mvadari) [reply snippets/src/getTransaction.ts:17]: We know that from the request that we sent, it won't be in binary format

**Pull request merged**